src/components/NewMessage/NewMessage.vue

```diff
@@ -148,9 +148,14 @@
 
 		<!-- Poll creation dialog -->
 		<NewMessagePollEditor v-if="showPollEditor"
+			ref="pollEditor"
 			:token="token"
 			@close="togglePollEditor" />
 
+		<PollDraftHandler v-if="showPollDraftHandler"
+			:token="token"
+			@close="togglePollDraftHandler" />
```

Comment:
Maybe in `SharedItemsTab` ?
It doesn't fit in both places 100% 🤔

Reply:
We don't have (yet) Shared Items tab in federated rooms
And NewMessage is a natural requirement for PollEditor -> Poll drafts

Reply:
Ah, I see your point. Let's keep it here then